pkgs/development/tools/jless/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jless";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "PaulJuliusMartinez";
+    repo = "jless";
+    rev = "v${version}";
+    sha256 = "sha256-gBqyo/N/qF6HCTUrSKNVLiL1fc/JTfip1kNpNCBzRT8=";
+  };
+
+  cargoSha256 = "sha256-PbX61RVbrI2kTuyXK+LhQdJDvNo3KjIQH5eBbL6iUBM=";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A command-line pager for JSON data.";
+    homepage = "https://github.com/PaulJuliusMartinez/jless";
+    license = licenses.mit;
+    maintainers = with maintainers; [ jfchevrette ];
+    mainProgram = "jless";
```

Comment:
Redundant as it matches `pname`.

```suggestion
```

Reply:
Removed. I read a bit more on meta.mainProgram and I understand it better now. Thanks for pointing it out!